### DISCUSSION TITLE
Does bevy comfortable for UI games?

### DISCUSSION THREAD

**Discussion by mrgzi** [bevyengine/bevy#10305] [Q&A]
I am using flutter for my crossword game. I will start to develop new crossword games and trying to figure out if bevy comfortable than flutter for ui games. Can you share your opinions?

**Comment by nicopap** [comment]
No. Bevy UI as of today is very difficult to work with. Come back in 6 months, I suspect the answer would be "Yes" by then.

**Reply by mrgzi** [reply]
Can you give more detail? Why it will be more comfortable than flutter?

**Reply by nicopap** [reply]
I misread you. I understood "Is bevy **as** comfortable as flutter".

I don't know flutter enough to give a comparison. Anyway, "comfortable" is subjective, and I can't speak for you. What I find comfortable is different from what you find comfortable.

There are several projects that want to improve bevy UI:

- One works with [dioxus](https://dioxuslabs.com/)
- another with [tauri](https://tauri.app/)
- And cart, the creator of bevy, is working on their own version, inspired by [godot](https://docs.godotengine.org/en/stable/about/faq.html#what-user-interface-toolkit-does-godot-use), see [this link](https://github.com/bevyengine/bevy/discussions/9538) for more details.

I can't tell if any of those projects will succeed. But it shows that people really want to make bevy UI great. With that kind of dedication, we'll see some nice UI in bevy soon.

**Reply by mrgzi** [reply]
Thanks for the answer. Actually, my purpose for switching to bevy is partly due to the problems caused by flutter. Junk animation is already a nightmare in Flutter. But more importantly, state management. I couldn't find a proper state management strategy in Flutter. What do you think about state management for Bevy UI games?
